.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+trim_trailing_whitespace = true
+indent_size = 4
+indent_style = space
+
+[Makefile]
+indent_style = tab
+indent_size = 4
+
+[*.{yml,json}]
+indent_size = 2
```

Comment:
Great @iMega, but You forgot a configuration for `[*.go]` files. Golang uses `indent_style = tab` and can't be configured as `[*]`.